src/bgen/CodeBlocks/BlockContainer.cs

```diff
@@ -0,0 +1,22 @@
+public class BlockContainer : CodeBlock {
+	public BlockContainer (int currentIndent) : base (currentIndent)
+	{
+		Indent = 0;
+	}
+
+	public override string Print ()
+	{
+		string s = string.Empty;
+
+		foreach (ICodeBlock block in Blocks) {
+			block.SetIndent (CurrentIndent);
+			s += block.Print ();
+		}
+		return s;
```

Comment:
How many Blocks can we get, if that is a lot of them, we should loot at using a StringBuilder.